content/docs/2.15/concepts/scaling-jobs.md

```diff
@@ -230,6 +230,11 @@ if (maxScale + runningJobCount) > maxReplicaCount {
 ```
 For more details,  you can refer to [this PR](https://github.com/kedacore/keda/pull/1227).
 
+**eager**
+When adopting the **default** strategy, you are likely to come into a subtle case where messages need to be consumed by spawning jobs but remain in the queue, even when there are available slots between `runningJobCount` and `maxReplicaCount`. The **eager** strategy comes to the rescue. It addresses this issue by utilizing all available slots up to the maxReplicaCount, ensuring that waiting messages are processed as quickly as possible.
+
```

Comment:
Can please we add here some example, so users understand the mode better? Something along the lines of the following comment: https://github.com/kedacore/keda/issues/5114#issuecomment-1782431472

Reply:
Done!

Reply:
Thanks!